Dockerfile

```diff
@@ -0,0 +1 @@
+FROM stain/jena-fuseki:latest
```

Comment:
Not sure it's supposed to be in the Dockerfile because I need my container to load the ontology (cf: https://hub.docker.com/r/stain/jena-fuseki/)
If I do it from the Dockerfile I'll be within a container and I don't want that (trying, load.sh script is failing)

Proposition: Wouldn't be better to create a ontologyUpdate.sh script and run it (travis or w/e) on the VM directly ? (Maybe I miss understand something)